app/javascript/src/components/Dashboard/Contacts/Table/constants.jsx

```diff
@@ -39,7 +39,9 @@ export const COLUMN_HEADERS = [
     render: () => (
       <Dropdown buttonStyle="text" icon={MenuHorizontal}>
         <li>{t("common.edit")}</li>
-        <li>{t("common.delete")}</li>
+        <li onClick={() => setIsDeleteOpen(prev => !prev)}>
+          {t("common.delete")}
+        </li>
       </Dropdown>
```

Comment:
Let's move this to a separate function prefixed with `render`.

Reply:
Fixed in [9d26873](https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/pull/16/commits/9d2687384782142f7195f006c99d2b6189262a8d)